tools/proto_format/BUILD

```diff
@@ -90,6 +90,8 @@ genrule(
         --xformed=$(location :xformed) \
         --protoprinted=$(location //tools/protoprint:protoprinted) \
     """ % PATH,
+    # TODO(phlax): make protoprint work without repo `PATH`
+    tags = ["no-remote"],
```

Comment:
this can be removed once #22004 lands

Reply:
That commit looks good, @htuch is out, hopefully @lizan can stamp that change first.

Reply:
@wrowe as this fixes local RBE for people i would suggest landing this first - the other PR makes quite a few ~unrelated changes, but i think getting this fix in is the priority

Reply:
This is now merged, so please update and we can merge this as well.